app/commands/get_user_pull_request_metrics.rb

```diff
@@ -30,21 +27,19 @@ def get_valid_pull_requests
     @github_user.pull_requests.where('gh_created_at > ?', limit_date).where.not(merged_by_id: nil)
   end
 
-  def get_pr_info(pull_request, review_req, time_delta)
+  def get_basic_pull_request_info(pull_request, review_request)
     { pr_title: pull_request.title, pr_created_at: pull_request.gh_created_at.to_s,
-      review_request_created_at: review_req.created_at.to_s,
-      time_delta: time_delta }
+      pr_merget_at: pull_request.gh_merged_at.to_s,
+      review_request_created_at: review_request.created_at.to_s }
   end
 
   def add_pr_data_to_hash(pull_request)
-    if review_req = pr_has_review_requests(pull_request)
-      time_delta = (review_req.created_at - pull_request.gh_created_at).to_i
+    if review_request = pr_has_review_requests(pull_request)
+      time_delta = (review_request.created_at - pull_request.gh_created_at).to_i
       unless time_delta > 1.week
-        @entry_count += 1
-        @total_time += time_delta
-        @review_requests_dic["pull_requests_information"][pull_request.id] = get_pr_info(
-          pull_request, review_req, time_delta
-        )
+        current_pull_request_info = get_basic_pull_request_info(pull_request, review_request)
+        current_pull_request_info.merge!(GetPullRequestReviewsInfo.for(pull_request: pull_request))
+        @review_requests_dic[:pull_requests_information][pull_request.id] = current_pull_request_info
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [101/100]